src/lib/wasm/MultipassStore.ts

```diff
@@ -574,7 +574,11 @@ class MultipassStore {
         if (multipass) {
             try {
                 let identityProfilePicture = await multipass.identity_picture(did)
-                profilePicture = identityProfilePicture ? this.to_base64(identityProfilePicture.data()) : ""
+                profilePicture = identityProfilePicture && identityProfilePicture.data 
+                ? (identityProfilePicture.data().length > 2 
```

Comment:
I think there was some possibility of the info returned having an "empty" image img/png:base64; or whatever lingering around still. If we can get a more reliable way to check for an empty image this could probably be removed and just rendered if the data is not null

Reply:
I do feel there is a better way to check to see if the data contains an image. Generally speaking, as long as its not empty or null, that should be enough.